Source/WebKit/GPUProcess/mac/com.apple.WebKit.GPUProcess.sb.in

```diff
@@ -226,7 +226,8 @@
         "kern.osproductversion"
         "kern.osrelease"
         "kern.ostype"
-        "kern.version")
+        "kern.version"
+        "vm.malloc_ranges") ;; <rdar://problem/105161083>
```

Comment:
I think we only need to allow this on startup, which means we can create a rule with the WebContentProcessLaunched state variable.

Reply:
Will do!